### ISSUE TITLE
Instance Norm vs. Batch Norm

### ISSUE THREAD

**Issue by kvttt** [liying-peng/MPGAN#1]
Hi, thanks for the great work! 
I have one clarification question. In the paper, you specified in Table 1 that in the feature extraction module, Instance Norm (indicated as IN) is used between layers. But in the [code](https://github.com/liying-peng/MPGAN/blob/7e91d1cc2fcd3d0636f1a8f342ed18651042eaa2/model/UnetGAN_percetual_loss.py#L85C1-L90C61) you provided, you have batch norm instead of instance norm enabled for the feature extraction module. I have also checked the pre-trained model provided by Models Genesis [here](https://github.com/MrGiovanni/ModelsGenesis/tree/master/keras). They also seem to be using batch norm instead of instance norm. Should I use BN or IN? Any help is appreciated!
